lasagne/utils.py

```diff
@@ -217,12 +217,16 @@ def compute_norms(array, norm_axes=None):
         set to a tuple listing all axes but axis 0. The former default is
         useful for working with dense layers, the latter is useful for 1D,
         2D and 3D convolutional layers.
+        Finally, in case `array` is a vector, norm_axes is set to an empty
```

Comment:
Minor nitpick: `norm_axes` should also be in single backticks, just like `array`.